prow/plugins.yaml

```diff
@@ -300,6 +300,7 @@ plugins:
   - cat
   - cla
   - dog
+  - golint
```

Comment:
isn't the lint plugin globally available now? @cjwagner

Reply:
This will enable it for the entire kubernetes-sig-testing org if that is what you mean, but that seems fine to me.

If you are asking if the golint plugin gets webhooks for all repos without having to be explicitly enabled, we don't have any mechanism for plugins to do that AFAIK.